pkg/util/helper/patch.go

```diff
@@ -32,3 +50,40 @@ func GenMergePatch(originalObj interface{}, modifiedObj interface{}) ([]byte, er
 
 	return patchBytes, nil
 }
+
+// GenReplaceFieldJSONPatch returns the RFC6902 JSONPatch array as []byte, which is used to simply
+// add/replace/delete certain JSON **Object** field.
+func GenReplaceFieldJSONPatch(path string, originalFieldValue, newFieldValue interface{}) ([]byte, error) {
+	if reflect.DeepEqual(originalFieldValue, newFieldValue) {
+		return []byte(`[]`), nil
+	}
+	if newFieldValue == nil {
+		return GenJSONPatch(JSONPatchOPRemove, "", path, nil)
+	}
+	// The implementation of “add” and “replace” for JSON objects is actually the same
+	// in “github.com/evanphx/json-patch/v5”, which is used by us and k8s.
```

Comment:
The presence of `us` word could be misleading.

`k8s` -> `K8s`

Reply:
Done.